x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/use_output_form.tsx

```diff
@@ -985,6 +989,21 @@ export function useOutputForm(onSucess: () => void, output?: Output, defaultOupu
               kibana_url: kibanaURLInput.value || null,
               proxy_id: proxyIdValue,
               ...shipperParams,
+              ssl: {
+                certificate: sslCertificateInput.value,
+                key: sslKeyInput.value || undefined,
+                certificate_authorities: sslCertificateAuthoritiesInput.value.filter(
+                  (val) => val !== ''
+                ),
+              },
+              ...(!sslKeyInput.value &&
+                sslKeySecretInput.value && {
+                  secrets: {
```

Comment:
there are more secrets in remote ES output, `ssl.key` should be added to the `secrets` var in line 965